### ISSUE TITLE
Error getting the Vagrant box of Kali_light_msf

### ISSUE THREAD

**Issue by marb08** [cliffe/SecGen#157]
I am trying to run some scenarios from the _**ctf**_ directory and I am experiencing that several .xml files, included the ones created by me, in particular the _**hackme.xml**_, produce the following error: 
```
The box failed to unpackage properly. Please verify that the box
file you're trying to add is not corrupted and that enough disk space
is available and then try again.
The output from attempting to unpackage (if any):

bsdtar: Ignoring malformed pax extended attribute
x ./box.ovf
bsdtar: Ignoring malformed pax extended attribute
x ./GNUSparseFile.0/box-disk001.vmdk
bsdtar: Ignoring malformed pax extended attribute
x ./Vagrantfile
bsdtar: Error exit delayed from previous errors.

Non-zero exit status...
Error provisioning VMs, destroying VMs and exiting SecGen.

```

The error always occurs just after the `attack_vm: Downloading: https://app.vagrantup.com/secgen/boxes/kali_light_msf/versions/1.0/providers/virtualbox.box
` part and I figured out that the archive file that is downloaded resulted to be empty (the archive manager output an empty archive error), even if it has a size of 1,8 GB. I also searched for a new download link in case of Vagrant changed it, but I can only reach this [path](https://app.vagrantup.com/secgen/boxes/kali_light_msf) without the possibility to download any archive.
I also noticed that Vagrant provides some information to modify the Vagrantfile about the import of the boxes, but even performing those steps, I got another error like this:
```
The box you're attempting to add doesn't support the provider
you requested. Please find an alternate box or use an alternate
provider. Double-check your requested provider to verify you didn't
simply misspell it.

If you're adding a box from HashiCorp's Vagrant Cloud, make sure the box is
released.

Name: secgen/kali_light_msf
Address: https://vagrantcloud.com/secgen/kali_light_msf
Requested provider: [:virtualbox]
```

I am running on a Debian 9 distro, with VirtualBox 5.0 and Vagrant 1.9.8 and I have 47GB of disk space available, so it is not a full disk problem.
If you want to replay the error just execute the following command:
`ruby secgen.rb -s scenarios/ctf/hackme.xml r`

**Comment by Brambopulos** [comment]
@cliffe slight bump for this guy, having the same issue on my team_project build. Thank you!

**Comment by Brambopulos** [comment]
Found the fix!

Update Vagrant to the latest version (more recent than what is available in the Ubu 16.04 **apt** repositories.

$ wget https://releases.hashicorp.com/vagrant/2.2.6/vagrant_2.2.6_x86_64.deb
$ sudo apt install ./vagrant_2.2.6_x86_64.deb

This allows you to unpack the file appropriately
